android/src/main/kotlin/com/spindance/flutter_weather_sensor_plugin/FlutterWeatherSensorPlugin.kt

```diff
@@ -1,35 +1,89 @@
 package com.spindance.flutter_weather_sensor_plugin
 
+import WeatherSensorService
```

Comment:
Not saying you have to use it, but I think it makes the interface on this side of the bridge a little nicer.

https://pub.dev/packages/pigeon

Reply:
I leaning towards not using pigeon mainly because I already wrote everything (unless you have a strong preference I switch), but it's good to know about for the future.

Reply:
Yeah don't feel like you have to switch over.  Just wanted to let you know it was out there as an option.